Sources/TestsCommon/ServerVersion.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+/// A struct representing a server version.
+public struct ServerVersion: Comparable, Decodable, CustomStringConvertible {
```

Comment:
similarly, this seems general enough that it can just be part of the common module.